core/tests/puppeteer-acceptance-tests/utilities/user/release-coordinator.ts

```diff
@@ -101,6 +200,82 @@ export class ReleaseCoordinator extends BaseUser {
       `Feature flag: "${featureName}" has been enabled successfully.`
     );
   }
+
+  /**
+   * Selects and runs a job.
+   * @param {string} jobName - The name of the job to run.
+   */
+  async selectAndRunJob(jobName: string): Promise<void> {
+    await this.type('.mat-input-element', jobName);
```

Comment:
Set them as constants. Ditto for the rest

Reply:
done.